src/pods/test-verb-forms/test-verb-forms.container.tsx

```diff
@@ -6,16 +6,17 @@ import { pickRandomVerb } from './test-verb-forms.business';
 import { useHistory } from 'react-router-dom';
 import { routes } from 'core/router';
 import { scoreContext } from 'core/score';
-
-// TODO: Move to const this could be configured maybe in profile context
-const totalQuestions = 15;
+import { settingsContext } from 'core/settings';
 
 export const TestVerbFormContainer = () => {
   const history = useHistory();
   const { selectedVerbs, verbCollection } = React.useContext(
     globalVerbsContext
   );
   const { setScore } = React.useContext(scoreContext);
+  const { userSettings } = React.useContext(settingsContext);
+  const totalQuestions = userSettings.numberQuestions;
```

Comment:
Mmmm this would evaluated on every render not sure if it's a good idea or if it would be a good idea just to use _userSettings.numberQuestions_ where applies (another option could be to use something like const [totalQuestions] = React.useState(userSettings.numberQuestions)